packages/flutter_image/lib/network.dart

```diff
@@ -53,6 +50,33 @@ class NetworkImageWithRetry extends ImageProvider<NetworkImageWithRetry> {
   /// fetch attempts failed.
   final FetchStrategy fetchStrategy;
 
+  /// HTTP Headers to add to the request.
+  ///
+  /// Keys from this map will be used as the header field names and the
+  /// values will be used as the header values. A list of header names can
+  /// be found at https://datatracker.ietf.org/doc/html/rfc7231#section-8.3
+  ///
+  /// Some headers are single valued, and for these, adding a value will
+  /// replace a previous value. If the value is a DateTime, an HTTP date
+  /// format will be applied. If the value is an Iterable, each element
+  /// will be added separately. For all other types the default
+  /// Object.toString method will be used.
+  ///
+  /// Header names are converted to lower-case. If two header names are
+  /// the same when converted to lower-case, they are considered to be
+  /// the same header, with one set of values.
+  ///
+  /// Example, adding Basic Authentication to requests.
```

Comment:
"For example, to add an authorization header to the request:"

Reply:
Updated sentence.